README.md

```diff
@@ -13,29 +13,29 @@ This plugin is tested on the below environment.
 
 ## Installation
 
-#### 1. Download source codes
+#### 1. Download source code
 
-Download whole project's source codes from [GitHub](https://github.com/nutti/UE4-Kdtree/archive/master.zip) and unzip them.
+Download whole project's source code from [GitHub](https://github.com/nutti/UE4-Kdtree/archive/master.zip) and unzip it.
 
 
-#### 2. Copy the plugin source codes
+#### 2. Move the plugin
 
-Copy the plugin source codes (`Kdtree` directory) into the plugin directory (`Plugin` directory) on your UE4 project.
+Move the plugin folder (`Kdtree` directory) into the plugins directory (`Plugins`) in your UE4 project folder.
```

Comment:
I think "into the \`Plugins\` directory" is much better than "into the plugins directory (\`Plugins\`)".